_pages/documentation/learning_gem5/part2/helloobject.md

```diff
@@ -203,15 +203,15 @@ file with the name "SConscript" in the `src/learning_gem5` directory. In
 this file, you have to declare the SimObject and the `.cc` file. Below
 is the required code.
 
-``` {.sourceCode .python}
+```
 Import('*')
 
 SimObject('HelloObject.py')
 Source('hello_object.cc')
 ```
 
-You can find the complete file
-here \<../\_static/scripts/part2/helloobject/SConscript\>.
+[//]: # You can find the complete file
+[//]: # here \<../\_static/scripts/part2/helloobject/SConscript\>.
```

Comment:
Do these links work on the new site?

Reply:
No but fixing that is on my TODO list. Shouldn't be too hard, I just commented them out in the meantime to avoid confusing/frustration.

Reply:
Makes sense :).